src/utils/immutability-helper.js

```diff
@@ -9,12 +9,11 @@ function mergeSingle(objA, objB) {
 export default {
 
   merge() {
-    const args = Array.prototype.slice.call(arguments, 0);
```

Comment:
There was no reason to clone args here. `arguments.length` and `arguments[i]` is safe usage, `Array#slice(arguments)` is not.